pkgs/by-name/go/google-chrome/package.nix

```
@@ -142,6 +142,18 @@ in stdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  passthru = {
+    updateScript = writeScript "update-google-chrome.sh" ''
+      #!/usr/bin/env nix-shell
+      #!nix-shell -i bash -p curl jq common-updater-scripts
+      res="$(curl -sL \
+        'https://versionhistory.googleapis.com/v1/chrome/platforms/linux/channels/all/versions/all/releases')"
```

That url is a bad idea, because it contains stable, beta and dev releases.

I guess you copied it from chromium's update.py, but there is a historic reason why we use it that does not apply here.

I'd highly recommend using 
```suggestion
        'https://versionhistory.googleapis.com/v1/chrome/platforms/linux/channels/stable/versions/all/releases')"
```
instead.

---

Thanks, you caught me cheating :)